UPGRADE-4.1.md

```diff
@@ -65,3 +65,14 @@ Workflow
  * Deprecated the `add` method in favor of the `addWorkflow` method in `Workflow\Registry`.
  * Deprecated `SupportStrategyInterface` in favor of `WorkflowSupportStrategyInterface`.
  * Deprecated the class `ClassInstanceSupportStrategy` in favor of the class `InstanceOfSupportStrategy`.
+ * Deprecated passing the workflow name as 4th parameter of `Event` constructor in favor of the workflow itself.
+ * [BCBREAK] The configuration of a place in XML has changed:
```

Comment:
can't we have a BC/FC XSD?

Reply:
I was not able to do that. My skill on with XSD are too limited. But I search and I could not find a way because the XML would become un-deterministic. But as I said, I really really doubt someone is using XML to configure workflows. I'm pretty sure it's dead code ;) 

But anyway, If someone could give me the XSD to matche both format, it would be nice.

Reply:
What if the place name was either configured using the attribute or the node content? This could be handle in the extension, right?

Reply:
Sorry I missed your comment. It might be possible, but it will be hard and it will add many code for almost nothing. I know nobody defining its workflow in XML. Obviously I don't know every workflow dev. But I really think it does not worth it.

Reply:
I removed the BC Break \o/